src/diffpy/labpdfproc/mud_calculator.py

```diff
@@ -68,33 +68,37 @@ def _compute_single_mud(x_data, I_data):
     """
     bounds = [
         (1e-5, x_data.max() - x_data.min()),  # diameter: [small positive value, upper bound]
-        (0, (x_data.max() - x_data.min()) / 2),  # slit width: [0, upper bound]
+        (0, (x_data.max() - x_data.min()) / 2),  # half slit width: [0, upper bound]
         (x_data.min(), x_data.max()),  # x0: [min x, max x]
         (1e-5, I_data.max()),  # I0: [small positive value, max observed intensity]
         (1e-5, 20),  # muD: [small positive value, upper bound]
-        (-10000, 10000),  # slope: [lower bound, upper bound]
+        (-100000, 100000),  # slope: [lower bound, upper bound]
```

Comment:
change slope bound. I think we want to support this for larger muD and larger slit widths.